android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1461,6 +1461,10 @@ protected void registerForTouch(final View touchable)
 		boolean clickable = true;
 		if (proxy.hasProperty(TiC.PROPERTY_TOUCH_ENABLED)) {
 			clickable = TiConvert.toBoolean(proxy.getProperty(TiC.PROPERTY_TOUCH_ENABLED), true);
+			//If touchEnabled is specifically set, enable or disable touch sound
+			touchable.setSoundEffectsEnabled(clickable);
+		} else {
+			touchable.setSoundEffectsEnabled(false);
```

Comment:
Alright. So let's introduce the property `<property name="ti.android.soundEffectsEnabled" type="bool" />` in the naming convention like `ti.android.bug2373.finishfalseroot` and merge this PR. @gstreetmedia can you apply this late change? If not, we would get an Android engineer of our team on it and merge this before. Thanks in advance!